src/posts/announce-2.0.0.Final.adoc

```diff
@@ -0,0 +1,168 @@
+---
+title: Announcing Thorntail 2.0.0.Final
+publishDate: 2018-06-27
+modifyDate: 2018-06-27
+author: The Thornail Team
+layout: post.jade
+---
+
+== Thorntail 2.0.0.Final
+
+The new name of WildFly Swarm is _Thorntail_! Enjoy the freedom of flying high like a Thorntail bird!
+
+=== What's New?
+
+* Did we say we have a new name: Thorntail?
+* We give up the `YYYY.MM.VV` versioning scheme and switched to a more convenient `major.minor.micro` versioning
+* MicroProfile
+** Thorntail now provides an implementation of the https://github.com/eclipse/microprofile-opentracing[MicroProfile OpenTracing specification]
+** Thorntail MicroProfile REST Client implementation introduces a non-portable feature - CDI Interceptors Support, see docs http://docs.wildfly-swarm.io/2.0.0.Final/#_cdi_interceptors_support[for more information and examples]
+
+++++
+<!-- more -->
+++++
+
+==== Migration Notes
+
+Migration from WildFly Swarm 2018.5.0 to Thorntail 2.0.0.Final is straightforward.
+Packages and class names have not changed.
```

Comment:
I'd also mention that configuration keys didn't change. And also the set of artifacts didn't change, only how to locate them.

Reply:
Good points.